dotnet-datetime/GetDatePartFromDateTime/GetDatePartFromDateTime/Program.cs

```diff
@@ -0,0 +1,36 @@
+﻿DateTime date1 = new DateTime(2022, 02, 14, 10, 40, 00); 
+DateTime date2 = new DateTime(2018, 10, 18, 11, 23, 34); 
```

Comment:
Use vars instead of explicit types whenever possible. Change in the entire solution.

Reply:
Some variables still use explicitly types in the test project.

Reply:
> Some variables still use explicit types in the test project.

Fixed